HerPublicWebsite/Views/Shared/_Layout.cshtml

```diff
@@ -75,7 +77,7 @@
             ServiceName = Constants.SERVICE_NAME,
             AssetsPath = assetsPath,
             HomepageUrl = "https://www.gov.uk",
-            ServiceUrl = Url.Action(nameof(StaticPagesController.Index), "StaticPages"),
+            ServiceUrl = WebHostEnvironment.IsProduction() ? Constants.SERVICE_URL : Url.Action(nameof(StaticPagesController.Index), "StaticPages"),

```

Comment:
(TBH it'd probably also be ok to have this as the gov.uk address in all environments, though I guess it's slightly inconvenient if you rely on it for testing)

Reply:
I found on SEA it was quite easy to accidentally end up on the live site while testing!